packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -876,7 +876,7 @@ void main() {
     final DebuggingOptions options = await command.createDebuggingOptions(true);
     expect(options.webLaunchUrl, 'http://flutter.dev');
 
-    final RegExp pattern = RegExp(r'^((http)?:\/\/)[^\s]+');
+    final RegExp pattern = RegExp(r'^((http|https)?:\/\/)[^\s]+');
```

Comment:
Can you add a test that passes a `web-launch-url` with https that passes the regex test?